src/js/controllers/amount.js

```diff
@@ -15,14 +13,17 @@ angular.module('copayApp.controllers').controller('amountController', function($
   });
 
   $scope.$on("$ionicView.beforeEnter", function(event, data) {
-
     $scope.isWallet = data.stateParams.isWallet;
     $scope.cardId = data.stateParams.cardId;
     $scope.toAddress = data.stateParams.toAddress;
     $scope.toName = data.stateParams.toName;
     $scope.toEmail = data.stateParams.toEmail;
     $scope.showAlternativeAmount = !!$scope.cardId;
     $scope.toColor = data.stateParams.toColor;
+    $scope.network = (new bitcore.Address($scope.toAddress)).network.name;
+    $scope.wallets = profileService.getWallets({
```

Comment:
$scope.network is used:

* line 36 > profileService.getWallets({onlyComplete: true, network: **$scope.network**})
* line 99 > feeService.getCurrentFeeValue(**$scope.network**, function(err, feePerKb) {...